pyshield/stencils/shallow_convection/__init__.py

```diff
@@ -0,0 +1,16 @@
+from ._config import ShallowConvectionConfig
+from .samfshalconv import ScaleAwareMassFluxShallowConvection
+from .state import SAMFShalConvState
+
+
+"""
+ShallowConvectionConfig: Configuration class for the shallow convection
```

Comment:
I'd advise to lean into this top-level docstring to describe the Shallow Convection scheme at large, including fortran and/or paper reference. We can then style the auto-doc to make it featured preeminently.